index.html 

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<head>
+	<title>Bubble Tea site</title>
+</head>
+<body>
+	<div class="topnav">
+		<img src="" alt="logo" style="width:600px; height:400px"/>
+		<a class="active" href=#home">Home</a>
+		<a href=#news">News</a>
+		<a href=#about">About</a>
+		<a href=#contact”>Contact</a>
+	</div>
+		
+    <div>
```

Comment:
You can see that these are not aligned.  All closing and opening tags need to be aligned, per level of the html hierarchy.